cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -117,8 +117,7 @@ type ClusterConfiguration struct {
 	CIImageRepository string
 
 	// UseHyperKubeImage controls if hyperkube should be used for Kubernetes components instead of their respective separate images
-	// DEPRECATED: As hyperkube is itself deprecated, this fields is too. It will be removed in future kubeadm config versions, kubeadm
-	// will print multiple warnings when set to true, and at some point it may become ignored.
+	// TODO: Remove this with v1beta2 https://github.com/kubernetes/kubeadm/issues/2459
```

Comment:
Should we remove this from internal types as well?
In the v1beta2 deprecation we already informed users that this will be ignored at a certain point...

Reply:
i've removed the field from the internal type.